### ISSUE TITLE
Graphics error

### ISSUE THREAD

**Issue by abyssalx** [Exzap/Cemu#7]
ay so I know full well that this is just only starting out and stuff. I just wanted to bring up the issue that I can load games into Cemu, and the fps counter starts counting, but no image is displayed in the window. Cemu doesn't freeze, and task manager shows CPU usage for Cemu. I have an ASUS Radeon HD 7770 (supports up to OpenGL 4.2) and an AMD A10-5800K (not overclocked) running Windows 10.

TL;DR, games load, image doesn't. Is it my hw config or the emulator? Anyway, I hope to see this project make some good progress in the near future! Wishing the dev team luck! :)

**Comment by deadair3210** [comment]
If I remember correctly the emulator is currently Nvidia only

**Comment by abyssalx** [comment]
ah alright, I get that o: thanks!
